packages/python/algokit_transact/pyproject.toml

```diff
@@ -1,26 +1,28 @@
 [build-system]
-requires = ["maturin>=1,<2"]
+requires = ["maturin>=1.5,<2"]
 build-backend = "maturin"
 
 [project]
 name = "algokit_transact"
 version = "0.1.0"
-requires-python = ">=3.7"
-dependencies = ["pynacl>=1.5.0"]
+requires-python = ">=3.8,<4.0"
+dependencies = ["pynacl>=1.5.0,<2.0.0", "cffi>=1.17.1"]
```

Comment:
It just occurred to me that `pynacl` is probably not an actual dependency... I believe it's just used for testing

Reply:
Removed